src/hooks/useJobProfiles.js

```diff
@@ -0,0 +1,31 @@
+/* istanbul ignore file */
```

Comment:
I see this comment on many files. Why should istanbul ignore things? This seems wrong.

Reply:
removed